runtest.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-TMPDIR=/dev/shm
+SHM_DIR=/dev/shm
+if [[ -d "${SHM_DIR}" ]]
+then
+  TMPDIR="${SHM_DIR}"
```

Comment:
What about still getting a TMPDIR in all cases on /tmp/<something>?
Things may fail otherwise? This is needed in https://github.com/WojciechMula/pyahocorasick/blob/377d371cb67e5d10385ecd608f5651d8c094a8c3/runtest.sh#L72 and a few other places.

Reply:
My bad, I omitted the default by mistake. Addressed in [c0d9151](https://github.com/WojciechMula/pyahocorasick/pull/172/commits/c0d91516e19a95292e82383368f2085e5991c5b0).